nutkit/protocol/cypher.py

```diff
@@ -515,6 +515,80 @@ def __eq__(self, other):
                    for attr in ("dtype", "data"))
 
 
+class CypherUnsupportedType:
+    r"""
+    A cypher unknown type.
+
+    :param name: The name of the unknown type.
+    :param minimum_protocol_major: The major bolt version needed for the type.
+    :param minimum_protocol_minor: The minor bolt version needed for the type.
+    :param message: An optional message to the user.
+    """
+
+    def __init__(
+        self, name, minimumProtocolMajor, minimumProtocolMinor, message=None
+    ):
+        self.name = str(name)
+        self.minimum_protocol_major = minimumProtocolMajor
```

Comment:
While we have decided to have 2 fields on Bolt level, are we still expecting the driver API to expose them together as a string? If yes, we should probably have the same here.